.github/workflows/go.yml

```diff
@@ -2,37 +2,48 @@ name: Go
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
```

Comment:
:+1: 

when does this run? Can we run this locally as well ? [nit] a doc if it can run locally

Reply:
VS code has a plugin that run this on `save` automatically.